### ISSUE TITLE
Update to python 3.7

### ISSUE THREAD

**Issue by jackwelty** [adplabs/adp-connection-python#10]
Hey this tripped me up as well, @ADPMarketplace please update your library.

_Originally posted by @jackwelty in https://github.com/adplabs/adp-connection-python/issues/7#issuecomment-618673995_

**Comment by jeremyehack** [comment]
@jackwelty Do you know what version of Python this lib actually runs on? I've been stepping my venv from 3.7 on down and can't seem to get it to import the package at all. 

Is it python2 only?

**Comment by jackwelty** [comment]
> @jackwelty Do you know what version of Python this lib actually runs on? I've been stepping my venv from 3.7 on down and can't seem to get it to import the package at all.
> 
> Is it python2 only?
> 
> @adplabs

Yep, it only works with python 2.x, but you can use [this fork](https://github.com/brandontksmith/adp-connection-python) with 3.x.

**Comment by Mahesh1240** [comment]
Hi
